chain-impl-mockchain/src/testing/e2e/fees/mod.rs

```diff
@@ -182,3 +182,91 @@ pub fn owner_delegates_fee() {
 
     ledger_verifier.total_value_is(&Value(expected_total_funds_after));
 }
+
+#[test]
+/// Verifies that after a transaction in a ledger without fees, the total funds do not change and
+/// the fee pots remain empty.
+fn transaction_without_fees() {
```

Comment:
Looks like those two methods are almost the same in term of the flow. the only diff is FEE value. Maybe we should introduce helper method with signature:

`fn verify_total_funds_after_transaction_with_fee(fee: u64)`

and then use it for two cases (non zero fees and zero fees).